packages/gestalt/src/TagData.css

```diff
@@ -0,0 +1,89 @@
+.tagOuterContainer {
```

Comment:
This is a _lot_ of additional CSS. Even styles composed from our utility classes are still additional classes to be included in the stylesheet. How much of these can be replaced by using fewer divs and more of our own components?